document/core/text/types.rst

```diff
@@ -138,6 +140,21 @@ Table Types
    \end{array}
 
 
+.. index:: event type, exception attribute, exception, function type
+   pair: text format; event type
+   pair: text format; exception attribute
+.. _text-eventtype:
+
+Event Types
+~~~~~~~~~~~
+
+.. math::
+   \begin{array}{llclll}
+   \production{event type} & \Teventtype &::=&
+     \text{(}~\text{exception}~~t^\ast{:}\Tvec(\Tparam)~\text{)} &\Rightarrow& \EXCEPTION~~[t^\ast] \to [] \\
```

Comment:
I'm not sure where this would actually be used in the text format, unless you define it as a synonym for a `typeuse`.

Reply:
Oh, this was just a remnant of having a more general event text format previously. I'm removing it.

Reply:
Done